examples/yc-batch-company-employees/src/cli/yc-company-employees.cli.ts

```diff
@@ -66,27 +69,16 @@ async function cli() {
       await confirm({ message: "Press enter once you have signed in", default: true });
 
       log.info("You can now close the browser tab for the live view. The extraction will continue in the background.");
-
-      // Update latest profile id
-      latestProfileId = ycSession.data.profileId;
-      log.info(`Latest profile id: ${latestProfileId}`);
-    }
-
-    if (!latestProfileId) {
-      throw new Error("No LinkedIn profile ID found, cannot continue");
     }
 
-    // Terminate session to persist profile
-    await airtop.terminateSession(ycSession.data.id);
```

Comment:
Didn't test this but in theory: If you plan on using the profile name on getEmployeesListUrls and getEmployeesProfileUrls batch operation (new sessions), you would need to save the profile and to do so, you would need to terminate.